### PR TITLE
qr: Add version 1.0.0

### PR DESCRIPTION
**Pull request by tpanj**
New package of utility.

### CODE REVIEW

**Review by tech189 — CHANGES_REQUESTED** [review]
```
"extract_dir": "\\",
```
Do not need this line - scoop will automatically extract the root directory of the zip file. Otherwise installation fails.

Add the following so scoop can get updates from github releases:
```
"checkver": "github",
    "autoupdate": {
        "url": "https://github.com/tpanj/qr/releases/download/v1.0.0/qr-$version-windows.zip"
    }
```

**Comment by tpanj** [comment]
> ```
> "extract_dir": "\\",
> ```
> Do not need this line - scoop will automatically extract the root directory of the zip file. Otherwise installation fails.
No, then this line is not needed.

> 
> Add the following so scoop can get updates from github releases:
> 
> ```
> "checkver": "github",
> "autoupdate": {
>     "url": "https://github.com/tpanj/qr/releases/download/v1.0.0/qr-$version-windows.zip"
> }
> ```
Thanks!

**Comment by niheaven** [comment]
@tpanj As the author of qr, could you please release sha256 checksum?

**Comment by rashil2000** [comment]
@tpanj As far as I understand, this repo was started just yesterday, and hence doesn't qualify for the Main criteria. 

This is why I insist on opening a Package Request first in the instructions template, which you didn't.

**Comment by tpanj** [comment]
> @tpanj As the author of qr, could you please release sha256 checksum?

You probably mean enclosed to qr...zip file?

**Comment by niheaven** [comment]
No, I mean a separate .sha256 file that contains hash, so Scoop's autoupdate could read hash from it instead of downloading and doing a local hash.

**Comment by tpanj** [comment]
> No, I mean a separate .sha256 file that contains hash, so Scoop's autoupdate could read hash from it instead of downloading and doing a local hash.

Like here [link](https://github.com/tpanj/qr/releases/download/v1.0.0/SHA256SUMS) ?

**Comment by niheaven** [comment]
Yes, and you'd better renamed this file to `qr-1.0.0-windows.zip.sha256` and changed au section of manifest to:

```json
    "autoupdate": {
        "url": "https://github.com/tpanj/qr/releases/download/v$version/qr-$version-windows.zip",
		"hash": {
			"url": "$url.sha256"
		}
    }
```

**Review by niheaven — APPROVED** [review]

File: bucket/qr.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "hash": "60a2e5bbf1c322e945b96c8e21995409877e2b48b2da92a2539756015c785ab6",
+    "bin": "qr.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/tpanj/qr/releases/download/v$version/qr-$version-windows.zip",
+         "hash": {
+            "url": "$url.sha256"
+         }
+    }
+
```
Comment (niheaven) [inline bucket/qr.json:16]:
```suggestion
    }
```

**Review by niheaven — CHANGES_REQUESTED** [review]
Remove that blank line

**Review by niheaven — APPROVED** [review]

**Pull request merged**